### PR TITLE
fix: ui bugs after #853

### PR DESCRIPTION
**Pull request by dozortsev**
## 🖼 Context

We introduced a few UI bugs in previous PR #853

## 🚀 Changes

See comments to changes.

## ✅ Checklist

- [x] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: packages/css/src/less/forms/bootstrap-forms.less [hunk 1]
```diff
@@ -773,6 +772,8 @@ input[type='checkbox'] {
   }
 }
 
+@import 'checkbox-radio.less';
```
Comment (dozortsev) [inline packages/css/src/less/forms/bootstrap-forms.less:775]: fix, reverting change that I made at https://github.com/transferwise/neptune-web/pull/853/files#diff-bbbddf816125427e1cf84aebf1c9046a2f226bf678f324f99dba0513b7f332f6R5

Screenshot 2021-03-24 at 10 33 31
Screenshot 2021-03-24 at 10 22 41

File: packages/css/src/less/input-groups.less [hunk 2]
```diff
@@ -220,6 +210,19 @@
...
+// stylelint-disable-next-line no-duplicate-selectors
+.input-group {
+  > :first-child .btn,
+  > :first-child .btn-group {
+    .border-right-radius(0);
+  }
+
+  > :last-child .btn,
+  > :last-child .btn-group {
+    .border-left-radius(0);
```
Comment (dozortsev) [inline packages/css/src/less/input-groups.less:222]: fix, reverting changes that I made here https://github.com/transferwise/neptune-web/pull/853/files#diff-b1ab0f56fa8ce6735d0b70858c1313adc02a8fa5ca7494d2092c01e65e19971cL200

Screenshot 2021-03-24 at 10 26 00

**Review by mikknarusk — APPROVED** [review]
🚀

File: packages/css/src/less/input-groups.less [hunk 3]
```diff
@@ -220,6 +210,19 @@
   .border-right-radius(@border-radius-base);
 }
 
+// stylelint-disable-next-line no-duplicate-selectors
```
Comment (dozortsev) [inline packages/css/src/less/input-groups.less:213]: rule https://stylelint.io/user-guide/rules/no-duplicate-selectors

**Pull request merged**